compiler/compiler.go

```diff
@@ -252,6 +252,13 @@ func Compile(pkgName string, machine llvm.TargetMachine, config *compileopts.Con
 
 	c.loadASTComments(lprogram)
 
+	// Forcibly preload special types.
+	runtimePkg := c.ir.Program.ImportedPackage("runtime")
+	c.getLLVMType(runtimePkg.Type("_interface").Type())
+	c.getLLVMType(runtimePkg.Type("_string").Type())
+	c.getLLVMType(runtimePkg.Type("hashmap").Type())
+	c.getLLVMType(runtimePkg.Type("channel").Type())
```

Comment:
Is this really necessary? I would expect these types to be included with the loop below.

Reply:
Unfortunately, some of these need to be pre-defined before they can be used by other runtime types.